nifi-mock/src/main/java/org/apache/nifi/util/MockPropertyValue.java

```diff
@@ -225,7 +225,7 @@ public String toString() {
 
     @Override
     public boolean isExpressionLanguagePresent() {
-        if (!expectExpressions) {
+        if (expectExpressions == null || !expectExpressions) {
```

Comment:
This fix (in whatever form it takes) should be as part of [NIFI-4590](https://issues.apache.org/jira/browse/NIFI-4590). As a temporary fix in my own PR #2260 , I put a try/catch around isExpressionLanguagePresent() with a comment about NIFI-4590. You don't have to do that specifically, but if you can get around changing the mock stuff in this PR, I can merge and add a reference to your workaround in NIFI-4590, to ask that your workaround be removed as part of that Jira.